### PR TITLE
Adds azure-pipelines.yml

### PR DESCRIPTION
**Pull request by MrFinchMkV**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    strategy:
+      matrix:
+        Release-Any:
+          Platform: 'Any CPU'
+          BuildConfiguration: 'Release'
+    workspace:
+      clean: outputs
+
+    steps:
+      - task: GitVersion@5
```
Comment (philipp-ranft) [inline azure-pipelines.yml:17]: Build version number differs from GUI based build. Maybe need to add a Gitversion.yml configuration file?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    pool: Cloud-Build
+    strategy:
+      matrix:
+        Release-Any:
+          Platform: 'Any CPU'
+          BuildConfiguration: 'Release'
+    workspace:
+      clean: outputs
+
+    steps:
```
Comment (philipp-ranft) [inline azure-pipelines.yml:13]: Can we add git cleaning to this pipeline?

**Review by philipp-ranft — CHANGES_REQUESTED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (MrFinchMkV) [reply azure-pipelines.yml:17]: Solved in: df5d64a

File: azure-pipelines.yml (hunk 2 above)
Comment (MrFinchMkV) [reply azure-pipelines.yml:13]: Solved in: a7a51db85969d53eab02299daa68d6355cb5f3e3

**Review by philipp-ranft — APPROVED** [review]

**Pull request merged**